events/tests/test_importer.py

```diff
@@ -20,7 +20,7 @@ def setUpClass(cls):
 
     def test_injest(self):
         importer = ICSImporter(self.calendar)
-        with open(EVENTS_CALENDAR) as fh:
+        with open(EVENTS_CALENDAR, encoding='utf-8') as fh:
```

Comment:
On my Windows 10 laptop, the result is the following:
```
>>> import locale
>>> locale.getpreferredencoding(False)
'cp1252'
```
As in a previous comment, I think it would be better for the time being to drop completely these changes, and only commit the ones about the event importer with tests.

Reply:
I've never tried to run python.org tests under Windows so feel free to send those changes as a separate PR :)